core/services/offchainreporting/keystore.go

```diff
@@ -100,7 +102,13 @@ func (ks KeyStore) GenerateEncryptedP2PKey() (p2pkey.Key, p2pkey.EncryptedP2PKey
 
 func (ks KeyStore) UpsertEncryptedP2PKey(k *p2pkey.EncryptedP2PKey) error {
 	err := ks.
-		Set("gorm:insert_option", "ON CONFLICT (pub_key) DO UPDATE SET encrypted_priv_key=EXCLUDED.encrypted_priv_key, updated_at=NOW()").
+		Set(
+			"gorm:insert_option",
+			"ON CONFLICT (pub_key) DO UPDATE SET "+
+				"encrypted_priv_key=EXCLUDED.encrypted_priv_key,"+
+				"updated_at=NOW(),"+
+				"deleted_at=null",
```

Comment:
This would be a little less unwieldy if you use a backtick string, which can contain newlines (and Postgres won't care about the newlines)
```suggestion
			`ON CONFLICT (pub_key) DO UPDATE SET
				encrypted_priv_key=EXCLUDED.encrypted_priv_key,
				updated_at=NOW(),
				deleted_at=null`
```